neo/rawio/plexonrawio.py

```diff
@@ -255,24 +259,29 @@ def _parse_header(self):
         else:
             chan_loop = range(nb_sig_chan)
         for chan_index in chan_loop:
-            h = slowChannelHeaders[chan_index]
-            name = h["Name"].decode("utf8")
-            chan_id = h["Channel"]
+            channel_headers = slowChannelHeaders[chan_index]
+            
+            # To avoid overflow errors when doing arithmetic operations on numpy scalars
+            np_scalar_to_python_scalar = lambda x: x.item() if isinstance(x, np.generic) else x
+            channel_headers = {key: np_scalar_to_python_scalar(channel_headers[key]) for key in channel_headers.dtype.names}
+            
+            name = channel_headers["Name"].decode("utf8")
+            chan_id = channel_headers["Channel"]
```

Comment:
Last question.

so below you called it dsp_channel_headers which was really nice. Any interest in changing this to `slow_channel_headers`, just so we make sure we never accidentally overwrite any variable. I think the explicit makes it cleaner and since we are already fixing up the variable naming might be worth the push?